### DISCUSSION TITLE
Is that a wrong command?

### DISCUSSION THREAD

**Discussion by Thespica** [apache/incubator-graphar#153] [General]
# Command

When I run the command in readme of cpp:

```
git submodule update --init
```

and get

```
fatal: not a git repository (or any of the parent directories): .git
```

So I tried change directory to GraphAr and it worked.

I consider the right commend should be what as follow?

```
$ git clone https://github.com/alibaba/GraphAr.git
$ cd GraphAr
$ git submodule update --init
$ cd cpp
```
If it's right, same as readme file in spark.

# A suggestion

It makes me confused about how to **run command and use it**, because there is no prompt in readme file or Getting Start. Finnally 
I find it in the cpp directory.

So I recommend to add a link that teachs how to run it in readme and Getting Start.

**Comment by lixueclaire** [comment]
Hi Thespica,
Thank you for your suggestion - we appreciate your input. Your use of the command was correct and we agree that our ReadMe could be more clear and intuitive for building steps and will work on refining it.

**Comment by lixueclaire** [comment]
PR #154 makes a change on this issue.
